packages/icon-build-helpers/src/builders/react/builder.js

```diff
@@ -114,16 +122,20 @@ export { Icon };
         // The length of this is determined by the number of directories from
         // our `outputOptions` minus 1 for the bundle type (`es` for example)
         // and minus 1 for the filename as it does not count as a directory jump
-        length: outputOptions.file.split('/').length - 2,
+        length: outputOptions.file.split(isWin ? '\\' : '/').length - 2,
       })
         .fill('..')
         .join('/');
 
+      let name = moduleName;
+      if (isWin && name.match(/^\d/)) {
+        name = '_' + moduleName;
```

Comment:
I believe the `moduleName` should already be sanitized as identifiers in JS also cannot start with a number similar to what I think the Windows requirement is above 👍

Reply:
Hmm... That does make sense, although that section did stall the build process,
Lemme look more into it.

Reply:
You're right, the logic that should be sanitizing this actually was missing names with a prefix that starts with a number.
Glad I looked at ti with fresh eyes  👌